application/values.yaml

```diff
@@ -514,3 +514,14 @@ certificate:
       create: false
       key: test_key
       name: test-creds
+
+# AlertmanagerConfig object for defining application specific alertmanager configurations
+alertmanagerConfig:
+  enabled: false
+  
+  # AlertmanagerConfig selectionLabels to specify label to be picked up by Alertmanager to add it to base config. Read more about it at [https://docs.openshift.com/container-platform/4.7/rest_api/monitoring_apis/alertmanager-monitoring-coreos-com-v1.html] under .spec.alertmanagerConfigSelector
+  selectionLabels:
+    alertmanagerConfig: "workload"
+
+  # AlertmanagerConfig spec, read details here [https://docs.openshift.com/container-platform/4.7/rest_api/monitoring_apis/alertmanagerconfig-monitoring-coreos-com-v1alpha1.html]
+  spec:
```

Comment:
please add some example spec in the comments

Reply:
exactly we should always have a sample spec